code/scripting/api/libs/mission.cpp

```diff
@@ -1337,6 +1331,9 @@ ADE_FUNC(addBackgroundBitmap,
 	}
 
 	sle.ang     = *orient.GetAngles();
+	if (!uses_correct_angles)
```

Comment:
Also wondering whether it's worth logging to the debug log which scripting function (deprecated or new) was called. Ditto for the functions for adding sun bitmaps.

Then again, does the fact that the old functions are deprecated result in a debug log message whenever they're called?

Reply:
Not a debug log message per se, but it will emit an error if the scripting function is called by a mod that requires a future version.